specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -8842,6 +8924,114 @@
         "modelAsString": true
       }
     },
+    "FlowAccessControlConfiguration": {
+      "type": "object",
+      "description": "The access control configuration.",
+      "properties": {
+        "triggers": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow triggers."
+        },
+        "contents": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow contents."
+        },
+        "actions": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow actions."
```

Comment:
The access control configuration for invoking workflow actions

Reply:
resolved in latest commit